src/art_blocks_plugin.h

```diff
@@ -42,6 +51,9 @@ typedef struct artblocks_parameters_t {
     uint8_t address_to[ADDRESS_LENGTH];
     uint8_t contract_address_sent[ADDRESS_LENGTH];
     char ticker_sent[MAX_TICKER_LEN];
+    uint8_t merkle_proof[PARAMETER_LENGTH];
+    uint8_t owned_nft_address[ADDRESS_LENGTH];
+    uint32_t owned_nft_id;
```

Comment:
This data structure has a maximum size of 160 bytes - we are currently using 128 of those bytes. Likely enough space to accommodate more minters, but we can shrink this data structure later if needed by reusing a shared buffer for different parameter types.

Reply:
got it, good callout, and we will be adding something like `ADDRESS_CONTRACT` after our minter suite refactor, so good to watch 🙏 a dev comment here might be a good idea so we don't forget about this 😄